lib/cosmos/tools/tlm_viewer/widgets/canvasvalue_widget.rb

```diff
@@ -68,6 +68,14 @@ def draw_widget(draw_context, on_state)
       raise 'Override draw_widget to make this class work!'
     end
 
+    def set_setting(setting_name, setting_values)
+      if setting_name =~ /^TLM_/
+        @settings["#{setting_name.to_s.upcase}_#{setting_values.join('_')}"] = setting_values
```

Comment:
Without doing this you could only ever have one TLM_AND and one TLM_OR. This deconflicts the hash key so you can AND and OR in multiple telemetry points.